app/kubernetes/react/views/beOnlyButton/index.tsx

```diff
@@ -1,27 +1,43 @@
+import { ReactNode } from 'react';
+
 import { FeatureId } from '@/react/portainer/feature-flags/enums';
 
 import { Button } from '@@/buttons';
 import { TooltipWithChildren } from '@@/Tip/TooltipWithChildren';
 
 interface Props {
   featureId: FeatureId;
+  heading: string;
+  message: string;
+  buttonText: string;
+  className?: string;
+  icon?: ReactNode;
 }
-export function YAMLReplace({ featureId }: Props) {
+
+export function BEOnlyButton({
```

Comment:
Or maybe it would be more accurate to prefix all components with `BETeaser`?   i.e. `BETeaserButton`